docs/front-end.md

```diff
@@ -8,25 +8,25 @@
-Currently this is achieved by relying on NSM that through a NSC container installs a VLAN interface into the particular frontend POD. The trunk interface (i.e. secondary network), the VLAN ID and the IP subnet of the VLAN network NSM will use to assign IP address to the external interface can be set during deployment to get consumed by the VLAN NSE.
+One way to achieve this is to rely on NSM which through a NSC container can install a VLAN capable interface into the particular frontend POD. The master interface residing in the host network namespace, the VLAN ID and the IP network NSM shall use to allocate IP address to the external interface must be configured to get consumed by the Remote VLAN NSE.  
+Alternatively, the external interface can be provided using Multus in which case no NSC or Remote VLAN NSE is required, and IP address allocation can be taken care of by a suitable IPAM CNI plugin (configured in the Network Attachment Definition).
+
 
 When started, the frontend installs src routing rules for each configured VIP address, then configures and spins off a [BIRD](https://bird.network.cz/) routing program instance providing for external connectivity. The bird routing suite is restricted to the external interface. The frontend uses [birdc](https://bird.network.cz/?get_doc&v=20&f=bird-4.html) for both monitoring and changing BIRD configuration.
 
-BGP protocol and Static+BFD setup are supported at the moment. Since they lack inherent neighbor discovery mechanism, the external gateway IP addresses must be configured during deployment time (or runtime once Meridio operator support is implemented).  
+BGP protocol and Static+BFD setup are supported at the moment. Since they lack inherent neighbor discovery mechanism, the external gateway IP addresses must be configured.
 In case of BGP a next-hop route for each VIP address gets announced by BGP to its external peer advertising the frontend IP as next-hop, thus attracting external traffic to the frontend. While from the external BGP peer a default next-hop route is expected that will be utilized by the VIP src routing to steer egress traffic.
 
 Both ingress and egress traffic traverse a frontend POD (not necessarily the same).
 
-Currently the frontend is collocated with the load balancer, hence reside in the same POD. A load balancer relies on the collocated frontend to forwarder egress traffic, and the other way around to handle ingress traffic. There's no direct communication between the two though.
-
-For setting external connectivity related parameters for Meridio refer to the vlan options in the values file and the [install guide](https://github.com/Nordix/Meridio/tree/master/docs/demo#meridio).
+Currently the frontend is collocated with the load balancer, hence reside in the same POD. A load balancer relies on the collocated frontend to forward egress traffic, and the other way around to handle ingress traffic. There's no direct communication between the two though.
```

Comment:
"There's no direct communication between the two though."  What does this mean? No direct control interaction? There will be payload communication, right?

Reply:
Yes, there's no control interaction between the two. Currently they are unaware of each other. Since they run in the same POD, one could argue that they do not exchange payloads either. They hook in to different parts of the packets processing.